pyproject.toml

```diff
@@ -155,23 +154,37 @@ manylinux-x86_64-image = "manylinux2014"
 manylinux-aarch64-image = "manylinux2014"
 musllinux-x86_64-image = "musllinux_1_1"
 # RUNNER_OS is a GitHub Actions specific env var; define it here so it works on Cirrus CI too
-environment = {RUNNER_OS="Linux"}
+[tool.cibuildwheel.linux.environment]
+RUNNER_OS="Linux"
+PKG_CONFIG_PATH="/project/.openblas"
```

Comment:
This appears to be GHA specific. Can the pc file be placed in a known location in CIBW_BEFORE_BUILD? What happens with the cirrus linux_aarch64 build which has a different directory structure to GHA?

Reply:
`/project` is a cibuildwheel convention, not a GHA one. It is a known path inside the docker image which is always used to build on linux with cibuildwheel.

Reply:
Worth a comment?